examples/todos/common/src/test/kotlin/org/reduxkotlin/examples/todos/TodosReducerSpec.kt

```diff
@@ -0,0 +1,111 @@
+package org.reduxkotlin.examples.todos
+
+import ch.tutteli.atrium.api.cc.en_GB.toBe
+import ch.tutteli.atrium.verbs.expect
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+object TodosReducerSpec : Spek({
+    describe("todos reducer") {
+        it("should handle AddToDo") {
+            expect(
+                todosReducer(
+                    listOf(), AddTodo(
```

Comment:
Also the listOf should probs just be `emptyList()` and we should use a named parameter to make this test more readable.

Reply:
done